model/Security/Vocabularies/CvssSeverityType.md

```diff
@@ -16,9 +16,9 @@ CvssSeverityType specifies the CVSS severity type, defined in the CVSS specifica
 
 ## Entries
 
-- critical: When a CVSS score is between 9.0 - 10.0
-- high: When a CVSS score is between 7.0 - 8.9
-- medium: When a CVSS score is between 4 - 6.9
-- low: When a CVSS score is between 0 - 3.9
-- none: When a CVSS score is 0
+- CRITICAL: When a CVSS score is between 9.0 - 10.0
```

Comment:
No, all our vocabularies are lowerCamelCase, and they get translated to RDF individuals.